CMakeLists.txt

```diff
@@ -74,6 +88,26 @@ if( VVDEC_ENABLE_ARM_SIMD )
   add_compile_options( ${FLAG_mfpu_neon} ${FLAG_mfloat_abi_hard} )
 endif()
 
+if( VVDEC_TARGET_ARCH STREQUAL "LOONGARCH64" )
+  if( VVDEC_ENABLE_LOONGARCH64_LSX_SIMD )
+    message( STATUS "LoongArch64 Lsx intrinsics enabled")
+    set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -mlsx")
+    set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -mlsx")
+  else()
+    message( STATUS "LoongArch64 lsx intrinsics disabled" )
+  endif()
+endif()
+
+if( VVDEC_TARGET_ARCH STREQUAL "LOONGARCH64" )
+  if( VVDEC_ENABLE_LOONGARCH64_LASX_SIMD )
+    message( STATUS "LoongArch64 Lasx intrinsics enabled")
+    set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -mlasx")
+    set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -mlasx")
+  else()
+    message( STATUS "LoongArch64 lasx intrinsics disabled" )
+  endif()
+endif()
+
```

Comment:
Please only add the `-mlasx -mlsx` flags to the SSE41 SIMD files (same way as `-mavx2` is handled). This can be done in `source/Lib/vvdec/CMakeLists.txt`

Reply:
I have made the modifications.

Reply:
When compiling 128-bit simd code, you need to add mlsx, and when compiling 256-bit simd code, you need to add mlasx. Based on the first solution, the relevant information regarding lasx has been removed.